src/js/cityDetails.js

```diff
@@ -0,0 +1,39 @@
+/* global document */
+
+const setUpDetails = (markerGroup, data) => {
+  const cityDetailsElement = document.querySelector(".city-details-popup");
+
+  // Clicking any city marker, updates HTML and makes popup visible
+  markerGroup.on("click", function (e) {
+    const cityState = e.sourceTarget.getTooltip().getContent();
+
+    cityInfo = createCityDetailsText(data, cityState);
```

Comment:
I'm surprised ESLint didn't complain. This should use `const` to assign a variable. Otherwise you get JavaScript weirdness

```suggestion
    const cityInfo = createCityDetailsText(data, cityState);
```